flow/core/params.py

```diff
@@ -44,8 +54,67 @@ def __init__(self, baseline=False):
         # traffic light xml properties
         self.__tls_properties = dict()
 
-        # all traffic light parameters are set to default baseline values
+        # all traffic light parameters are set to default baseline values.
+        # NOTE: This overrides all customization
```

Comment:
Could you define more clearly what is meant by customization?

Reply:
done